content/courses/intro-to-ada/chapters/privacy.rst

```diff
@@ -247,11 +247,11 @@ summarizes this:
 | Part of a child package       | Access to the private part of |
 |                               | its parent's specification    |
 +===============================+===============================+
-| Specification: public part    |                               |
+| Specification: public part    | No                            |
 +-------------------------------+-------------------------------+
-| Specification: private part   | X                             |
+| Specification: private part   | Yes                           |
 +-------------------------------+-------------------------------+
-| Body                          | X                             |
+| Body                          | Yes                           |
```

Comment:
This isn't really a "minor fix", but rather an editorial change. Having explicit "no"s and "yes"s might be unequivocal, but it also makes the tables more difficult to read.

I assume you have a problem with the "X", which could potentially be interpreted as "doesn't apply" in some contexts. If that's the case, you could replace them by check marks, which always mean "yes" — for that, use the `|check|` syntax from `isopub.txt`.

Reply:
> be interpreted as "doesn't apply"

This was exactly my case. `|check|` should work indeed.